.gitignore

```diff
@@ -3,6 +3,13 @@
 /releases/temp/
 NOTES.md
```

Comment:
remove this too? seems like this is totally misplaced.

Reply:
done